pyspecProcScripts/phasing.py

```diff
@@ -468,6 +469,7 @@ def filter_ranges(B, A):
     d[direct, 0] *= 0.5
     d.ft(direct)
     return d
+    return peakrange
```

Comment:
just remove this line.  It's actually not doing anything, anyways.  And, given that you set the `"peakrange"` property above, it's now accessible as `d.get_prop("peakrange")`

Reply:
removed